neural_compressor/torch/algorithms/static_quant/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023 Intel Corporation

```

Comment:
```suggestion
# Copyright (c) 2024 Intel Corporation

```
The recently added files should use `2024`. Please check it in other files as well.